### DISCUSSION TITLE
改变一行文字中的数字或者百分比值的颜色

### DISCUSSION THREAD

**Discussion by fallenleavesguy** [hizzgdev/jsmind#530] [General]
请问一下,内容是一行文字，文字中有+10%, -10%这样的文字，我需要根据不同的值设置不同的颜色，如红绿色，突显文字的变化方向，这个可以实现吗

**Comment by hizzgdev** [comment]
不太懂你的意思啊。不过有 api 可以设置节点的颜色字体和背景。https://github.com/hizzgdev/jsmind/blob/master/docs/zh/3.operation.md
你可以在点击事件中获取节点对应的 node_id，这样就可以使用这些 api 了。

**Reply by fallenleavesguy** [reply]
我的意思大概是这样的。节点里一行文字: 今天涨幅 +10% ，+10%这个数字希望可以单独修改颜色，例如，正数用红色，负数用绿色，0用灰色

**Reply by hizzgdev** [reply]
节点里的内容可以使用 html, 所以你应该有办法做到的 😄

**Reply by fallenleavesguy** [reply]
不是canvas的吗？里面竟然可以用html，真的吗

**Reply by fallenleavesguy** [reply]
好吧，没细看，我不以为是全部内容都是画布里的呢，点赞

**Reply by hizzgdev** [reply]
另外前两天更新了一个新的版本 [0.7.6](https://github.com/hizzgdev/jsmind/releases/tag/v0.7.6) ，支持自定义 render 逻辑。
这是相关的文档：[view.custom_node_render](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/2.options.md) ，你也可以试试这个。
